test/kubernetes_unittest.cc

```diff
@@ -0,0 +1,70 @@
+#include "../src/kubernetes.h"
+#include "../src/updater.h"
+#include "../src/configuration.h"
+#include "../src/instance.h"
+#include "gtest/gtest.h"
+
+namespace google {
+
+class KubernetesTest : public ::testing::Test {
+  protected:
+   MetadataUpdater::ResourceMetadata GetNodeMetadata(
+       const KubernetesReader& reader, const json::Object *node,
+       Timestamp collected_at, bool is_deleted) const
+       throw(json::Exception) {
+      return reader.GetNodeMetadata(node, collected_at, is_deleted);
+   }
+};
+
+TEST_F(KubernetesTest, GetNodeMetadata) {
+  Configuration config(std::stringstream(
+    "KubernetesClusterName: 'TestClusterName'\n"
```

Comment:
Single quotes behave weirdly in YAML. Since these are all bare words, let's just not quote them at all?..